planex/cmd/clone.py

```diff
@@ -76,10 +84,7 @@ def main(argv=None):
 
         if args.jenkins:
             print 'echo "Cloning %s"' % pin.url
-            print CHECKOUT_TEMPLATE.substitute(url=pin.url,
-                                               branch=pin.commitish,
-                                               checkoutdir=checkoutdir,
-                                               credentials=args.credentials)
+            clone_jenkins(pin.url, checkoutdir, pin.commitish, args.credentials)
```

Comment:
I would be effect free in clone_jenkins, returning the fragment, and move the print back to this line.